### ISSUE TITLE
Incompatible browser warning

### ISSUE THREAD

**Issue by plasmoduck** [Chocobozzz/PeerTube#4306]
I always get Incompatible browser warning when using a webkit browser called [surf](https://surf.suckless.org/)
This browser actually works fine.

**Comment by Chocobozzz** [comment]
Hello,

Please fill the bug template and provide console logs

**Comment by plasmoduck** [comment]
I'm talking about this:

PeerTube
Incompatible browser

We are sorry but it seems that PeerTube is not compatible with your web browser.

Please try with the latest version of Mozilla Firefox.

If you think this is a mistake, please report it.

**Comment by Chocobozzz** [comment]
Yes I understood. This message is displayed because there was a JS error. It's the reason why I need the web browser console logs

**Comment by plasmoduck** [comment]
Oh I think because surf blocks is

**Comment by ROBERT-MCDOWELL** [comment]
FYI I got this JS error on my side
Timestamp: 8/7/2021 8:55:28 AM
Error: SyntaxError: expected expression, got keyword 'import'
Source File: https://xxx/client/en-US/main.edd299a39fa834d242c4.js
Line: 3, Column: 1137015
Source Code:
onst a=(h.N.apiUrl||window.location.origin)+n.script;return import(a).then(t=>t.register({registerHook:i,registerVideoFi

**Comment by Chocobozzz** [comment]
If this is the error in surf, then it's the same answer than https://github.com/Chocobozzz/PeerTube/issues/4311#issuecomment-896628797

**Comment by ROBERT-MCDOWELL** [comment]
@Chocobozzz
are dynamic imports are new from version 3.0? because this site conf.tube uses version 3.0.1 and can be shown nicely in palemoon and mypal.
